nativeApp/src/lib/ui/forms/TimeForm/index.tsx

```diff
@@ -93,7 +95,7 @@ const TimeForm: FC<Props> = (props) => {
         mode="time"
         minDate={startTime}
       />
-      <Label>{`${volunteer || 'Member'} volunteered for`}</Label>
+      <Label>{getTimeLabel(forUser, volunteer)}</Label>
```

Comment:
It was verbally agreed in a meeting that we would trial this and if users come back with saying that it's an issue we will.

This was an on the spot decision though, it may be worth listing all the small changes introduced in this app for the team to refer back to incase any reasoning changes.

Reply:
covered in #398